testing/scenario_app/android/README.md

```diff
@@ -1,102 +1,364 @@
-# Scenario App: Android Tests
+# Scenario App: Android Tests and Test Runner
 
-As mentioned in the [top-level README](../README.md), this directory contains
-the Android-specific native code and tests for the [scenario app](../lib). To
-run the tests, you will need to build the engine with the appropriate
-configuration.
+End-to-end tests and test infrastructure for the Flutter engine on Android.
 
-For example, for the latest `android` build you've made locally:
+> [!IMPORTANT]
+> There are several **known issues** with this test suite:
+>
+> - [#144407](https://github.com/flutter/flutter/issues/144407): "Newer Android"
+>   tests are missing expected cropping and rotation.
+> - [#144365](https://github.com/flutter/flutter/issues/144365): Tests that use
+>   `ExternalTextureFlutterActivity` sometimes do not render.
+> - [#144232](https://github.com/flutter/flutter/issues/144232): Impeller Vulkan
+>   sometimes hangs on emulators.
+> - [#144352](https://github.com/flutter/flutter/issues/144352): Skia Gold
+>   sometimes is missing expected diffs.
+
+---
+
+This test suite was [originally written in 2019](https://github.com/flutter/engine/pull/10007)
+with a goal of:
+
+> \[being\] suitable for embedders to do integration testing with - it has no
+> dependencies on the flutter_tools or framework, and so will not fail/flake
+> based on variances in those downstream.
+
+Unfortunately, the Android side of the test suite was never fully operational,
+and the tests, even if failing, were accidentally be reported as passing on CI.
+In 2024, as the team got closer to shipping our new graphics backend,
+[Impeller](https://docs.flutter.dev/perf/impeller) on Android, it was clear that
+we needed a reliable test suite for the engine on Android, particularly for
+visual tests around external textures and platform views.
+
+So, this package was revived and updated to be a (more) reliable test suite for
+the engine on Android. It's by no means complete
+([contributions welcome](#contributing)), but it did successfully catch at least
+one bug that would not have been detected automatically otherwise.
+
+_Go forth and test the engine on Android!_
```

Comment:
I don't think this history-in-prose should get top billing.  I think a `Timeline` section lower down would make more sense.  We can keep the guiding principles.

Something like:
```
## Timeline
- 2019 Originally implemented: lorem ipsom
- 2024 Revamped to work for impeller.

Reply:
Done!